docker/env/redis-cache.env

```diff
@@ -0,0 +1 @@
+REDIS_PASSWORD=t4Ph722qJ5QHeQ1qfu36
```

Comment:
mmm that was meant as REDIS_CACHE_PASSWORD but it was used like this and which is consistent in netbox.env file so I think it is fine as it is because when I change the name then the container starts crashing.